app/controllers/listing_controller.rb

```diff
@@ -4,6 +4,11 @@ def index
     @listing_detail_props = { assetPaths: static_asset_paths }
   end
 
+  def how_to_apply
+    @how_to_apply_props = { assetPaths: static_asset_paths }
+    render 'how_to_apply' if ENV['HOW_TO_APPLY_PAGE_REACT'].to_s.casecmp('true').zero?
```

Comment:
I think we could. I lean towards using the current pattern and creating a ticket to switch to unleash

Reply:
I agree with alula 

We should stop using environment variables and use unleash instead in this implementation instead of creating more work for ourselves

We can handle the redirect on the rails side in the controller - https://edgeapi.rubyonrails.org/classes/ActionController/Redirecting.html

Reply:
Can probably redirect to the listing details page rather than the home page though

Reply:
Made those changes and updated the review instructions